airbyte-webapp/.storybook/preview.ts

```diff
@@ -3,6 +3,7 @@ import { addDecorator } from "@storybook/react";
 import { withProviders } from "./withProvider";
 
 import "!style-loader!css-loader!sass-loader!../public/index.css";
+import "../src/scss/global.scss";
```

Comment:
Just a thought: `global.scss` and `index.css` probably belong together, they are basically both global CSS.

Reply:
All the styles in `index.css` are currently meant to be applied even if JS doesn't work, thus they can't be loaded in the global.scss. Whether we really need this is though another question, but I'd leave that for a separate change.